migrations/0067_tlv2_request_id_not_null.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE time_limited_v2_order_creds ALTER COLUMN request_id SET NOT NULL;
```

Comment:
At the moment of writing, there have been no rows with `request_id` `NULL` in the table (both Staging and Production).